wire-compiler/src/main/java/com/squareup/wire/CommandLineOptions.java

```diff
@@ -202,10 +203,10 @@ private ServiceFactory loadServiceFactory(String className) throws WireException
     return Arrays.asList(arg.substring(flagLength).split(","));
   }
 
-  public String protoPath() {
-    String result = protoPath;
+  public List<String> protoPaths() {
+    List<String> result = protoPaths;
     if (result == null) {
```

Comment:
I'm guessing this is going to fail, cause the value will be an empty list instead of null